wagtail/tests/settings_ui.py

```diff
@@ -5,4 +5,19 @@
 
 DATABASES["default"]["NAME"] = "ui_tests.db"  # noqa
 
-WAGTAIL_EXPERIMENTAL_FEATURES = {"slim-sidebar"}
+INSTALLED_APPS += [  # noqa
+    "pattern_library",
+]
+
+TEMPLATES[0]["OPTIONS"]["builtins"] = ["pattern_library.loader_tags"]  # noqa
+
+PATTERN_LIBRARY = {
+    # Groups of templates for the pattern library navigation. The keys
+    # are the group titles and the values are lists of template name prefixes that will
+    # be searched to populate the groups.
+    "SECTIONS": (("components", ["wagtailadmin/shared"]),),
```

Comment:
django-pattern-library requires this being set so it knows which templates it can render – I’ve just gone ahead with this setup for the time being, but there is nothing preventing us from using it for all templates.